src/main/java/it/gov/pagopa/reward/notification/controller/NotificationControllerImpl.java

```diff
@@ -84,6 +89,28 @@ public Mono<Page<RewardExportsDTO>> getExportsPaged(String organizationId, Strin
                 .switchIfEmpty(Mono.just(Page.empty(pageable)));
     }
 
+    @Override
+    public Mono<ExportSummaryDTO> getSingleExportSummary(String organizationId, String initiativeId, String exportId) {
+        return exportDetailService.getExportSummary(organizationId, initiativeId, exportId);
+    }
+
+    @Override
+    public Flux<ExportDetailDTO> getSingleExport(String organizationId, String initiativeId, String exportId, Pageable pageable, ExportDetailFilter filters) {
+        return exportDetailService.getSingleExport(organizationId, initiativeId, exportId, pageable, filters);
+    }
+
+    @Override
+    public Mono<ExportPageDTO> getSingleExportPaged(String organizationId, String initiativeId, String exportId, Pageable pageable, ExportDetailFilter filters) {
+        return exportDetailService
+                .getSingleExportPaged(organizationId, initiativeId, exportId, pageable, filters)
+                .switchIfEmpty(exportDetailService.getExportDetailEmptyPage(pageable));
+    }
+
+    @Override
+    public Mono<RefundDetailDTO> getSingleRefund(String organizationId, String initiativeId, String eventId) {
+        return exportDetailService.getSingleRefundDetail(organizationId, initiativeId, eventId);
```

Comment:
what if empty? I don't remember what's the behaviour, but let's test and check ;)